cmd/launcher/main.go

```diff
@@ -27,6 +28,9 @@ func main() {
 		os.Exit(1)
 	}
 
+	sentry.Init()
```

Comment:
this confused me until I checked the import. maybe we can rename this to `errorReporter` instead?

Reply:
Good point. I'll rename

Reply:
Good point. Renamed to `reporting`. Go [recommends](https://go.dev/blog/package-names) using all lowercase package names. LMK if you have a better name in mind